Source/swiftlint/Helpers/LintOrAnalyzeCommand.swift

```diff
@@ -26,6 +26,11 @@ enum LintOrAnalyzeMode {
 
 struct LintOrAnalyzeCommand {
     static func run(_ options: LintOrAnalyzeOptions) async throws {
+        if let workingDirectory = options.workingDirectory {
+            if !FileManager.default.changeCurrentDirectoryPath(workingDirectory) {
+                throw Issue.couldNotChangeToWorkingDirectory(path: workingDirectory)
```

Comment:
So originally this `throw` produced the following eventual output:

`Error: couldNotChangeToWorkingDirectory(path: "Source/SwiftLintt")`

because the thrown error eventually gets passed to `MessageInfo(error: Error, type: ParsableArguments.Type)` from the `swift-argument-parser`. Ideally it would get caught there by

```
      case let error as LocalizedError where error.errorDescription != nil:
        self = .other(message: error.errorDescription!, exitCode: .failure)
```

but it was not, because `Issue`'s implementation of `errorDescription` (`var errorDescription: String`) does not match the `LocalizedError` protocol requirement (`var errorDescription: String?`).

I've fixed that, but because of the way the thrown error is now printed, we get 

`Error: warning: Could not change working directory to 'Source/SwiftLintt'.`

One way around that would just be to brute-force:

```
if !FileManager.default.changeCurrentDirectoryPath(workingDirectory) {
    Issue.couldNotChangeToWorkingDirectory(path: workingDirectory).print()
    exit(2)
}
```

Another possibility would be to remove our `warning: ` and `error: ` prefixes from `errorDescription`, but to add them back in `Issue.print()` - most usages look like that would be ok, but I'm not sure how confident I am that all of them would be.

Reply:
Switched to `SwiftLintError.usageError`, which produces: `Error: Could not change working directory to 'Source/SwiftLintt'

I would kind of expected. throwing an `Issue` to work though.